app/goal_routes.py

```diff
@@ -0,0 +1,74 @@
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.goal import Goal
+from app.models.task import Task
+from app import db
+
+
+goal_bp = Blueprint("goal", __name__, url_prefix="/goals")
+
+def validate_model(cls, model_id):
+        try:
+            model_id=int(model_id)
+        except:
+            abort(make_response({"message":f"{cls.__name__}{model_id} invalid"}, 400))
+        model =  cls.query.get(model_id)
```

Comment:
Quick note about formatting, it's a good idea to read back through your PRs to catch small formatting mistakes like the lack of space around the assignment on line 11 and the extra space on the right side of the assignment on this line (14). There are a number of formatters but the one you might want to look at first has a VS Code integration called AutoPEP8.